documentation/api/query/tutorial-pql.markdown

```diff
@@ -90,6 +90,17 @@ only accept unencrypted traffic from `localhost`.
 This requires that you specify a certificate (issued by the same CA PuppetDB
 trusts), a private key, and a CA certificate.
 
+### Querying with puppet code
+
+The PuppetDB terminus includes the `puppetdb_query` function, which can be used
+to query PuppetDB from within a Puppet manifest. For example,
+
+    $last_node_query = 'nodes{order by report_timestamp desc limit 1}'
```

Comment:
might be helpful to note you can use the AST syntax too, `$last_node_query = [ "from", "nodes", <stuff> ]`